### PR TITLE
0.2.11 release notes

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
This PR adds release notes for 0.2.11

### CODE REVIEW

**Review by tlambert03 — APPROVED** [review]

File: docs/release/release_0_2_11.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+Highlights
+**********
+- Allow Points display properties to be set by point properties (#895)
+- Python 3.8 support (#878)
```
Comment (kne42) [inline docs/release/release_0_2_11.rst]: I would also include #915 here

File: docs/release/release_0_2_11.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+- Add _add_layer_data method and tests (#909)
+- Add Python 3.8 to PyPI tags (#917)
+- Relocate existing tests for appropriate discoverability. (#918)
+
+Bugfixes
+********
+- Fix clim popup position in floating widgets (#869)
+- Change autodevdoc script to use new git reference (#876)
+- Fix invalid instructions in setup.py (#877)
+- Clean up setup.py (#880)
```
Comment (kne42) [inline docs/release/release_0_2_11.rst]: I think this should be in Improvements since it's not fixing bugs but rather improving readability

File: docs/release/release_0_2_11.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+- Represent color as Nx4 array in Points  (#782)
+- Qt/Vispy connection & lambda refactor (#859)
+- Add PyQt5 tests to Linux and OSX CI (#867)
+- Bump tests to python 3.8, general tests fix (#878)
+- Improve slider step precision based on data range (#884)
+- Allow Points display properties to be set by point properties (#895)
+- Refactor add_* methods (#897)
+- Add _add_layer_data method and tests (#909)
+- Add Python 3.8 to PyPI tags (#917)
+- Relocate existing tests for appropriate discoverability. (#918)
```
Comment (kne42) [inline docs/release/release_0_2_11.rst]: I think this belongs in Bugfixes since arguably, the tests should never have been discoverable to begin with

File: docs/release/release_0_2_11.rst [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+Bugfixes
+********
+- Fix clim popup position in floating widgets (#869)
+- Change autodevdoc script to use new git reference (#876)
```
Comment (kne42) [inline docs/release/release_0_2_11.rst]: is this even a bug? I'm not sure how relevant it is

File: docs/release/release_0_2_11.rst [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+************
+- Represent color as Nx4 array in Points  (#782)
+- Qt/Vispy connection & lambda refactor (#859)
+- Add PyQt5 tests to Linux and OSX CI (#867)
+- Bump tests to python 3.8, general tests fix (#878)
+- Improve slider step precision based on data range (#884)
+- Allow Points display properties to be set by point properties (#895)
+- Refactor add_* methods (#897)
+- Add _add_layer_data method and tests (#909)
+- Add Python 3.8 to PyPI tags (#917)
```
Comment (kne42) [inline docs/release/release_0_2_11.rst]: not sure how relevant this is

**Review by kne42 — COMMENTED** [review]
I think we should add another `Support` section or something since I feel like a lot of these don't fit under the provided categories.

IMO `Improvements`/`Bugfixes` should solely relate to changes that directly impact the user in terms of functionality so that it's easy for users to find changes that are relevant to them when they're skimming the release  notes.

The `Support` category would include PRs related to:
- tests
- CI
- documentation
- small dependency changes
- any other change that does not impact the public-facing functionality significantly

File: docs/release/release_0_2_11.rst [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+Highlights
+**********
+- Allow Points display properties to be set by point properties (#895)
```
Comment (jni) [inline docs/release/release_0_2_11.rst]: Here I would add a short paragraph to highlight this one, as it's a pretty major new API feature. e.g.,

```rest
- point face color and edge color are now settable as a property
in a columnar data table, mapped using a colormap (continuous values)
or a color cycle (categorical values). See
`this example <https://github.com/napari/napari/blob/master/examples/add_points_with_properties.py>`_ 
for syntax details.
```

**Review by jni — APPROVED** [review]
@sofroniewn awesome! Not bad at all for what was supposed to be a minor bug fix release!!!

I've only made a minor suggestion, but otherwise I approve!

File: docs/release/release_0_2_11.rst (hunk 6 above)
Comment (jni) [reply docs/release/release_0_2_11.rst]: btw markdown doesn't render ReST hyperlinks very well, here's the syntax reference fyi:

http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks

Comment (jni) [reply docs/release/release_0_2_11.rst]: (**edit**: I've edited the above comment so that the syntax is properly shown)

File: docs/release/release_0_2_11.rst (hunk 1 above)
Comment (sofroniewn) [reply docs/release/release_0_2_11.rst]: done

File: docs/release/release_0_2_11.rst (hunk 2 above)
Comment (sofroniewn) [reply docs/release/release_0_2_11.rst]: good catch

File: docs/release/release_0_2_11.rst (hunk 3 above)
Comment (sofroniewn) [reply docs/release/release_0_2_11.rst]: Good idea

File: docs/release/release_0_2_11.rst (hunk 4 above)
Comment (sofroniewn) [reply docs/release/release_0_2_11.rst]: I've moved this to improvements. I like having every PR listed here for the sake of being comprehensive

Comment (sofroniewn) [reply docs/release/release_0_2_11.rst]: Have moved to new `support` category

File: docs/release/release_0_2_11.rst (hunk 5 above)
Comment (sofroniewn) [reply docs/release/release_0_2_11.rst]: As noted above I like this list to be comprehensive, but I have created a new support category as you suggested so it does not become to hard to parse

**Comment by sofroniewn** [comment]
I have updated this PR taking into account the requests of @kne42 (including adding a Support category) and @jni adding more detail to the new points properties API.

If this looks ok I'd like to merge so I can start making the new release

**Comment by jni** [comment]
Looks great! :tada:

**Comment by sofroniewn** [comment]
I'm going to merge now and start making the release candidate. @kne42 if there are things you want changed we can change them while people test the release candidate.

File: docs/release/release_0_2_11.rst [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+- Add _add_layer_data method and tests (#909)
+
+Bugfixes
+********
+- Fix clim popup position in floating widgets (#869)
+- Bump tests to python 3.8, general tests fix (#878)
+- Fix points selection (#902)
+- Move benchmarks under napari directory but not distributed (#913)
+- Patch py3.8 on windows (#915)
+- Fix osx py3.6 tests (#916)
```
Comment (kne42) [inline docs/release/release_0_2_11.rst]: this belongs under `Support`

File: docs/release/release_0_2_11.rst [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+Highlights
+**********
+- Point face color and edge color are now settable as a property in a columnar data table, mapped using a colormap (continuous values) or a color cycle (categorical values). See `this example <https://github.com/napari/napari/blob/master/examples/add_points_with_properties.py>`_ for syntax details.
+- Python 3.8 is now supported.
+
+Improvements
+************
+- Represent color as Nx4 array in Points  (#782)
+- Qt/Vispy connection & lambda refactor (#859)
+- Clean up setup.py (#880)
```
Comment (kne42) [inline docs/release/release_0_2_11.rst]: support

File: docs/release/release_0_2_11.rst [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+- Improve slider step precision based on data range (#884)
+- Allow Points display properties to be set by point properties (#895)
+- Refactor add_* methods (#897)
+- Add _add_layer_data method and tests (#909)
+
+Bugfixes
+********
+- Fix clim popup position in floating widgets (#869)
+- Bump tests to python 3.8, general tests fix (#878)
```
Comment (kne42) [inline docs/release/release_0_2_11.rst]: support

**Review by kne42 — APPROVED** [review]

File: docs/release/release_0_2_11.rst (hunk 7 above)
Comment (sofroniewn) [reply docs/release/release_0_2_11.rst]: done

File: docs/release/release_0_2_11.rst (hunk 8 above)
Comment (sofroniewn) [reply docs/release/release_0_2_11.rst]: done

File: docs/release/release_0_2_11.rst (hunk 9 above)
Comment (sofroniewn) [reply docs/release/release_0_2_11.rst]: done

**Comment by sofroniewn** [comment]
Thanks for looking at this again @kne42! I have moved those three requested lines to the support section. If there is nothing else, I will then merge this, but I'll let it sit for a few minutes in case anyone wants to flag something else
